src/transformers/modeling_t5.py

```diff
@@ -1027,6 +1027,16 @@ def forward(
 
 @add_start_docstrings("""T5 Model with a `language modeling` head on top. """, T5_START_DOCSTRING)
 class T5ForConditionalGeneration(T5PreTrainedModel):
+    keys_to_ignore_at_load = [
```

Comment:
the varname `keys_to_ignore_at_load` suggests that we won't load them.
Suggested name: `keys_that_can_be_missing_without_concern`

Reply:
Right now, the keys are also ignored if the checkpoint has them but not the model (like in T5 case) which is why it's named like this. If we remove the layernorm bias keys in T5, we can change the name (though I'd prefer something shorter, seven words is a bit too verbose ;-) I think `authorized_missing_keys` is enough).